gui-wm/hyprland/hyprland-0.19.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 Aisha Tammy <floss@bsd.ac>
+# Distributed under the terms of the ISC License
+
+EAPI=8
+
+inherit toolchain-funcs meson
+
+DESCRIPTION="Dynamic tiling Wayland compositor that doesn't sacrifice on its looks."
+HOMEPAGE="https://github.com/hyprwm/Hyprland"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hyprwm/Hyprland"
+else
+	MY_PV="${PV}beta"
+	SRC_URI="https://github.com/hyprwm/Hyprland/releases/download/v${MY_PV}/source-v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}"
```

Comment:
Instead of a custom `src_unpack` set this variable correctly - `S="${WORKDIR}"/hyprland-source`